javascript-data-model/order-history.js

```diff
@@ -0,0 +1,81 @@
+var orderFour = [
+  {
+    name: 'GameCube Controller'
+  },
+  {
+    name: 'The Art of Sql'
+  }
+];
+
+var orderHistory = [
+  {
+    name: 'Javascript for impatient programmers',
+    author: 'Dr. Axel Rauschmayer',
+    orderDate: 'August 4, 2020',
+    ifDelivered: true,
+    deliveryDate: 'August 8,2020',
+    subtotal: 31.55,
+    grandTotal: 34,
+    recipient: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    returnWindowOpen: false,
+    returnWindowDate: 'Sep 7, 2020',
+    reviewWritten: false
+  },
+  {
+    name: 'The Timeless Way of Building',
+    author: 'Christopher Alexander',
+    orderDate: 'July 19, 2020',
+    ifDelivered: true,
+    deliveryDate: 'July 20,2020',
+    subtotal: 41.33,
+    grandTotal: 44.53,
+    recipient: 'JS Masher',
+    orderNumber: '113-9984268-1280257',
+    returnWindowOpen: false,
+    returnWindowDate: 'Aug 19, 2020',
+    reviewWritten: false
+  },
+  {
+    name: 'Gamecube Controller Adapter',
+    orderDate: 'July 4, 2020',
+    ifDelivered: true,
+    deliveryDate: 'July 7,2020',
+    subtotal: 15.98,
+    grandTotal: 17.22,
+    recipient: 'JS Masher',
+    orderNumber: '114-2875557-9059409',
+    returnWindowOpen: false,
+    returnWindowDate: 'Aug 5, 2020',
+    reviewWritten: false
+  },
+  {
+    orderFourContent: [
+      {
+        name: 'Gamecube Controller - Super Smash Bros. Edition',
+        orderDate: 'July 3, 2020',
+        ifDelivered: true,
+        deliveryDate: 'July 5,2020',
+        subtotal: 94.95,
+        returnWindowOpen: false,
+        returnWindowDate: 'Aug 4, 2020',
+        reviewWritten: false
+      },
+      {
+        name: 'The Art of Sql',
+        author: 'Stephane Faroult',
+        orderDate: 'July 3, 2020',
+        ifDelivered: true,
+        deliveryDate: 'July 5,2020',
+        subtotal: 33.99,
+        returnWindowOpen: false,
+        returnWindowDate: 'Aug 4, 2020',
+        reviewWritten: false
+      }
+    ],
```

Comment:
Notice how this entry has arrays for values, but the rest of your entries have strings for values. This is not a consistent data model, which will cause issues if you ever tried to use this data. Make sure that your format matches for all orders.